src/lava/lib/dnf/connect/connect.py

```diff
@@ -39,12 +42,15 @@ def connect(
     ops : list(AbstractOperation), optional
         list of operations that describes how the connection between
         <src_op> and <dst_ip> will be created
+    connection_class : Class of the connecting process (Dense/Sparse), optional
```

Comment:
I think for consistency it might be better to have:
`connection_class: type, optional`
Because technically, `Dense` and `Sparse` are of type `type`.